src/data/finset/basic.lean

```diff
@@ -1394,7 +1394,7 @@ by congr
 
 section classical
 open_locale classical
-/-- The following instance allows us to write `{ x ∈ s | p x }` for `finset.filter s p`.
+/-- The following instance allows us to write `{ x ∈ s | p x }` for `finset.filter p s`.
```

Comment:
I think we usually don't use spaces around braces when using "set-builder" notation like this (the spaces are only for braces which surround tactics). Do you mind editing the extra spaces out of the rest of the docstring as well?
```suggestion
/-- The following instance allows us to write `{x ∈ s | p x}` for `finset.filter p s`.
```